services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_states.py

```diff
@@ -30,13 +33,15 @@
 # mapping container states into 4 categories
 # For all avaliable containerstates SEE
 # https://github.com/moby/moby/blob/master/container/state.go#L140
-CONTAINER_STATUSES_FAILED: set[str] = {"restarting", "dead", "paused"}
-CONTAINER_STATUSES_STARTING: set[str] = {"created"}
-CONTAINER_STATUSES_RUNNING: set[str] = {"running"}
-CONTAINER_STATUSES_COMPLETE: set[str] = {
+CONTAINER_STATUSES_FAILED: set[str] = {
+    "restarting",
+    "dead",
+    "paused",
     "removing",
     "exited",
```

Comment:
I am not sure these 2 additional state are actually failed ones. ```removing``` is an active state right?
and ```exited``` does not mean it is failed, only if the exit code is non zero.
You might need to either rename that set or maybe explain why these 2 states are suddenly part of it?

Reply:
I think a rename of the CONTAINER_STATUES_FAILED is required.
I'd say that these should be called undesired states. If the containers are in these states, something went wrong with the container.

Reply:
what about ```exited```? this one is also correct in a proper shutdown

Reply:
The containers are expected to be running, all other states here are unexpected.